pages.pl/osx/wc.md

```diff
@@ -0,0 +1,24 @@
+# wc
+
+> Zlicza linie, słowa, i bajty
+> Więcej informacji: <https://www.gnu.org/software/coreutils/wc>.
+
+- Policz linie w pliku
+
+`wc -l {{plik}}`
```

Comment:
Could you apply this change to `path/to/file` in the main page?
```suggestion
`wc -l {{ścieżka/do/pliku}}`
```

Reply:
Yes, I can, but the osx English TLDR page of `wc` also says `file`, and not `path/to/file`. 

If you always want the longer version suggesting that files can have a path, then this is not the only place where I translated `{{file}}` to `{{plik}}`.

Reply:
yeah, we are making a migration if we refresh an old page.